javalib/src/main/scala/java/lang/String.scala

```diff
@@ -831,8 +831,61 @@ final class _String()
     preprocessed.toLowerCase()
   }
 
-  @inline
-  def toLowerCase(): _String = toCase(Character.toLowerCase)
+  @inline def toLowerCase(): _String = {
```

Comment:
This should probably not be `@inline` anymore.
```suggestion
  def toLowerCase(): _String = {
```